udfs/remote_udfs/nlp/call_nlp/main.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from google.cloud import language_v1
+from google.api_core.retry import Retry
+
+# https://cloud.google.com/functions/docs/bestpractices/tips#use_global_variables_to_reuse_objects_in_future_invocations
+client = language_v1.LanguageServiceClient()
+
+def remote_vertex_ai(request):
```

Comment:
Why is it called remote_vertex_ai?
shouldn't this be "remote_sentiment_analysis"?

Reply:
Because there is another step I removed that used to call another vertex AI endpoint.
The "mode" is where you specify the sentiment analysis / nlp processing.

Reply:
I'll leave it as remote_vertex_ai unless you have a strong opinion otherwise.

Reply:
I changed it to remote_call_nlp "choosing" the execution of plaintext with additional comments.